src/Cache.php

```diff
@@ -29,11 +29,12 @@ class Cache
      * Returns false on any error (typically, inability to write to file), true
      * on success.
      *
-     * @param  string $filename
-     * @param  \Laminas\Server\Server $server
+     * @param string $filename
+     * @param Server $server
      * @return bool
```

Comment:
The same here: the exact information is already included in the parameter list and in the return value of the method.

Reply:
Even if there is a description and also a @throws tag?

Reply:
No no, omit only the lines with `@param` and `@return` if the information is already present in parameter list and the return value. Descriptions and `@throws` are good and an important information.

(Maybe that wasn't clear enough but I marked only the lines 32 - 34, not the entire DocBlock. I didn't mean to mislead you.)

Reply:
Probably misunderstood you. Thanks for the explanation.

I went over the codebase and removed all redundant @param and @return tags. I check this point as resolved. If I've overlooked something, please let me know.